### DISCUSSION TITLE
Battery status icon

### DISCUSSION THREAD

**Discussion by thabit** [maca88/SmartBikeLights#76] [General]
I have a Bontraeger ION 200 RT lamp. It is new and fully charged. However, the battery status bar never shows 100%. 

With the rear light, it works.
![IMG_3376](https://github.com/maca88/SmartBikeLights/assets/4584023/b3ae4677-a9b8-4f0e-bc9e-e3f9a3307e57)

**Comment by maca88** [comment]
This is a common issue with Bontrager lights, my Flare RT suffers the same issue. After charging, you need to do a factory reset by holding down the power button for 15 seconds (release after the second flash). You need to do the factory reset twice as the first one will disable ANT & Bluetooth (no blue light).

**Comment by thabit** [comment]
This helped. ThanksAm 23.07.2023 um 20:34 schrieb maca88 ***@***.***>:﻿
This is a common issue with Bontrager lights, my Flare RT suffers the same issue. After charging, you need to do a factory reset by holding down the power button for 15 seconds (release after the second flash). You need to do the factory reset twice as the first one will disable ANT & Bluetooth (no blue light).

**Comment by thabit** [comment]
Hi ,

I am now using an Edge 840 and want to set the 2 data fields (front and
back light). I configured the settings as follows and posted in the
ConnectIQ app > Bike Lights Control #2, Settings under lights:
#4587520,196641::1##6291461,1409482753::1##3,2:Ion 200
RT:0:16777215!2,:-1,Off:0!1,Low:5#3,2:Flare
RT:0:16777215!2,:-1,Off:0!1,All-Day Flash:8#0::#0:0#B4062#2#1#0#0

However, in the data fields, there are not only these 2 options shown, but
I can select different between many different ones. is the configuration
not loaded?

Thanks

Am So., 23. Juli 2023 um 20:34 Uhr schrieb maca88 ***@***.***

**Comment by maca88** [comment]
Hi,

Bike Lights Control is not a data field but a widget. You need to Install Smart Bike Lights data field instead.

**Comment by thabit** [comment]
Ok thanks
